### PR TITLE
Background

### PR DESCRIPTION
**Pull request by jarrodmillman**
Stefan and I will work on this PR for the next few days.  Closes PR #26

### CODE REVIEW

File: scipy-1.0/paper.tex [hunk 1]
```diff
@@ -118,7 +122,302 @@ \section*{Introduction}
...
+custom-built hardware was feasible.
+
+With a rich programming environment and a numerical array object in
+place, the time was ripe for the development of a full scientific
+software stack.  Many of the first efforts were driven by graduate
+students and post-doctoral researchers, but soon attracted the
+attention of senior researchers.
+% Paul Dubois, CDAT
+% https://mail.python.org/pipermail/numpy-discussion/2001-May/000645.html
+Paul Dubois at Lawrence Livermore National Laboratories took over the
```
Comment (tylerjereddy) [inline scipy-1.0/paper.tex]: Laboratories -> Laboratory

File: scipy-1.0/paper.tex [hunk 2]
```diff
@@ -118,7 +122,302 @@ \section*{Introduction}
...
+% https://mail.python.org/pipermail/scipy-user/2003-March/001455.html
+
+In 2000, STScI ported their entire IDL-based analysis pipeline, IRAF,
+to Python.  And it was here that a small crisis developed around the
+Python numerical array container.  Numeric, the original array package, was
+suitable for small arrays, but not for the large images processed by
+STScI.  With the Numeric maintainer's blessing, the decision was made
+to write NumArray, a library that could handle data on a different
+scale.  Unfortunately, NumArray proved inefficient for small arrays,
+facing the community with a rather unfortunate choice.  In 2005,
```
Comment (tylerjereddy) [inline scipy-1.0/paper.tex]: facing -> presenting, probably

File: scipy-1.0/paper.tex [hunk 3]
```diff
@@ -118,7 +122,302 @@ \section*{Introduction}
...
+\begin{verbatim}
+    2001: first SciPy release
+    2005: transition to NumPy
+    2007: creation of scikits
+    2008: spatial module and first Cython code added
+    2010: move to a 6-monthly release cycle
+    2011: development moves to GitHub
+    2011: Python 3 support
+    2012: add a sparse graph module and unified optimization interface
+    2012: remove of maxentropy
```
Comment (tylerjereddy) [inline scipy-1.0/paper.tex]: "removal of"

**Comment by stefanv** [comment]
Thanks, Tyler, I fixed up those typos.

**Comment by stefanv** [comment]
OK, I think we're done for now.  We'll do another round of checking closer to final publication.

**Comment by rgommers** [comment]
Yes, that makes sense. Besides the one or two unfinished sentences at the end, this looks quite polished. In it goes, thanks Stefan and Jarrod!

**Pull request merged**